source/config.js

```diff
@@ -299,6 +302,7 @@ const argv = yargs
     'numRefsToShow',
     'Number of refs to show on git commit bubbles to limit too many refs to appear.'
   )
+  .describe('diffTabSize', 'Specify tab size for diff')
```

Comment:
```suggestion
  .describe('diffTabSize', 'Specify tab size for diff in css units')
```

I would just add it would be nice to say it is css units otherwise I'd have though it is tab size in spaces

Reply:
Based on the [docs](https://www.w3schools.com/cssref/css3_pr_tab-size.asp) it actually work with number of space characters

Reply:
Based on the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/tab-size) it actually work with number of space characters.

Length values are not supported in some browsers (Safari)

Reply:
oh good to know thanks,t hen you can discard my comment

Reply:
We could mention that the value should be number of spaces to be clear